integration_tests/commands/async/append_test.go

```diff
@@ -0,0 +1,100 @@
+package async
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+)
+
+func randString(n int, charset string) string {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = charset[r.Intn(len(charset))]
+	}
+	return string(b)
+}
```

Comment:
No need for randomly generated keys in a test. It makes it harder to debug.

Reply:
This is more of a fuzz/stress test, I have removed it.